frontend/src/components/misc/GuideWrapper.jsx

```diff
@@ -0,0 +1,272 @@
+import React from "react";
+import Joyride from "react-joyride";
+import { Outlet, useNavigate } from "react-router-dom";
+import { useMount } from "react-use";
+import { useGuideContext } from "../../contexts/GuideContext";
+
+export default function GuideWrapper() {
+  const { guideState, setGuideState } = useGuideContext();
+  const navigate = useNavigate();
+
+  const steps = [
+    {
+      target: "#home__bgImg",
+      content: (
+        <div id="guidebox">
+          <h3>Guide</h3>
+          <p>
+            Welcome to IntelOwls Guide for First Time Visitors!, for further
+            questions you could check out our <a href="/">docs</a> or reach us
+            out on <a href="/">intelowls slack channel</a>
+          </p>
+        </div>
+      ),
+      disableBeacon: true,
+    },
+    {
+      target: "#Dashboard_title",
+      content: (
+        <div id="guidebox">
+          <h3>Dashboard</h3>
+          <p>See previous job details here with charts and more</p>
+        </div>
+      ),
+      disableBeacon: true,
+    },
+    {
+      target: "#Dashboard_timepicker",
+      content: (
+        <div id="guidebox">
+          <h3>Filter</h3>
+          <p>Filter by time to get details about previous jobs</p>
+        </div>
+      ),
+      disableBeacon: true,
+    },
+    {
+      target: "#Analyzers",
+      content: (
+        <div id="guidebox">
+          <h3>Plugins</h3>
+          <p>
+            Analyzers are the most important plugins in IntelOwl. They allow to
+            perform data extraction on the observables and/or files that you
+            would like to analyze.
+          </p>
+        </div>
+      ),
+      disableBeacon: true,
+    },
+    {
+      target: "#pluginconfigbutton",
+      content: (
+        <div id="guidebox">
+          <h3>Plugin Configurations</h3>
+          <p>Write up your own plugin configs</p>
+          <p>
+            Note: Your plugin configuration overrides your organizations
+            configuration (if any).
+          </p>
```

Comment:
```suggestion
          <p>Write up your own plugin configuration!</p>
          <p>
            Note: Some plugins work out-of-the-box, while others requires to be configured (with API keys for instance).
          </p>
```
I think this info is too specific.